test/spec-bin.sh

```diff
@@ -7,11 +7,7 @@
 #   ./spec-bin.sh <function name>
 #
 # Instructions:
-#   test/spec-bin.sh download     # Get the right version of every tarball
-#   test/spec-bin.sh extract-all  # Extract source
-#   test/spec-bin.sh build-all    # Compile
-#   test/spec-bin.sh copy-all     # Put them in _tmp/spec-bin
-#   test/spec-bin.sh test-all     # Run a small smoke test
+#   test/spec-bin.sh make-all     # Download, extract, compile, and test all shells
```

Comment:
I like the comment as is because it tells people to run steps separately... but you can add a note about the "all-in-one" function too